worker.js

```diff
@@ -63,40 +85,51 @@ var getBucketList = function (cb) {
 var readBuckets = function (tasks) {
   console.info('--- Started indexing all buckets ---');
   async.parallel(tasks,
```

Comment:
4 seems reasonable (along with a comment explaining why), to achieve an effective limit of 20 (see below).